[Release v0.6.0] Editor x Simulator v2 x Sentry

arnaudlewis opened the pull request:
Contains all 3 main features of the v0.6.0:
- Editor
- Sentry
- Simulator v2

danigo99 committed:
Fix double braces in uuid field

bapmrl committed:
Publish

 - @slicemachine/client@1.1.16-dev-react-18.2
 - @slicemachine/core@1.1.16-dev-react-18.2
 - @slicemachine/init@1.1.16-dev-react-18.2
 - slice-machine-ui@0.5.3-dev-react-18.2

hypervillain committed:
setupModal variable tabs len

hypervillain committed:
safeguard unsupported framework

hypervillain committed:
use theme value + default fontWeight

bapmrl committed:
add Router prop back

bapmrl committed:
Merge branch 'dev-react-18' of github.com:prismicio/slice-machine into dev-react-18

bapmrl committed:
update package-lock.json files

bapmrl committed:
Publish

 - @slicemachine/client@1.1.16-dev-react-18.3
 - @slicemachine/core@1.1.16-dev-react-18.3
 - @slicemachine/init@1.1.16-dev-react-18.3
 - slice-machine-ui@0.5.3-dev-react-18.3

hypervillain committed:
display yarn command

hypervillain committed:
add yarn to excerpt

hypervillain committed:
Merge pull request #724 from prismicio/design-review-sim-2

setupModal variable tabs len

MarcMcIntosh committed:
refactor: move updating mocks to `api/state`

MarcMcIntosh committed:
refacor: update mocks, rename things

MarcMcIntosh committed:
Merge branch 'ex246/custom-type-mocks' into exs240/save-mocks

bapmrl committed:
remove @svgr/webpack from package.json

bapmrl committed:
Merge branch 'dev-react-18' of github.com:prismicio/slice-machine into dev-react-18

bapmrl committed:
update package-lock.json files

bapmrl committed:
Publish

 - @slicemachine/client@1.1.16-dev-react-18.4
 - @slicemachine/core@1.1.16-dev-react-18.4
 - @slicemachine/init@1.1.16-dev-react-18.4
 - slice-machine-ui@0.5.3-dev-react-18.4

MarcMcIntosh committed:
chore: use t.strict rather than t.type

MarcMcIntosh committed:
refactoring: rename updateMocks to validateOrReplceMocks

MarcMcIntosh committed:
Merge branch 'ex246/custom-type-mocks' into exs240/save-mocks

arnaudlewis committed:
Merge branch 'dev-react-18' into mocks

arnaudlewis committed:
Merge branch 'mocks' into ex246/custom-type-mocks

arnaudlewis committed:
fix dnd type issue

arnaudlewis committed:
fix drop type issue in tabzone

MarcMcIntosh committed:
chore: update package-lock files

MarcMcIntosh committed:
refactor: quick fixes for dragEndEvents and ensureDnDDestination

Commit:
add new dependency when starting a next project

MarcMcIntosh committed:
Revert "refactor: quick fixes for dragEndEvents and ensureDnDDestination"

This reverts commit fd1596a7f91624179ffe2e59877227100434aecc.

arnaudlewis committed:
Merge pull request #725 from prismicio/fix/dnd-type-issue

Drag n Drop type issue

arnaudlewis committed:
Merge branch 'dev-react-18' into mocks

arnaudlewis committed:
Merge branch 'mocks' into ex246/custom-type-mocks

arnaudlewis committed:
Merge pull request #675 from prismicio/ex246/custom-type-mocks

Ex246/custom type mocks

MarcMcIntosh committed:
Merge branch 'mocks' into exs240/save-mocks

 Conflicts:
	cypress/integration/slices/00-create.spec.js
	packages/client/package-lock.json
	packages/client/package.json
	packages/core/package-lock.json
	packages/init/package-lock.json
	packages/init/package.json
	packages/slice-machine/.depcheckrc
	packages/slice-machine/components/Simulator/index.tsx
	packages/slice-machine/jest.config.js
	packages/slice-machine/package-lock.json
	packages/slice-machine/package.json

MarcMcIntosh committed:
chore: fix depcheck

MarcMcIntosh committed:
chore: run prettier

MarcMcIntosh committed:
Update dependency: next-router-mock@0.8.0

next-router-mock
  * slice-machine-ui: ^0.6.7 → 0.8.0

MarcMcIntosh committed:
test(sm-ui): mock mediaWatcher for simulator

MarcMcIntosh committed:
bug: set light mode on theme provider

MarcMcIntosh committed:
chore: update package-lock

MarcMcIntosh committed:
refactor(slice-mock): not sure why the ci is failing here

MarcMcIntosh committed:
refactor: remove unused import

MarcMcIntosh committed:
chore: undo last changes, error persists :/

MarcMcIntosh committed:
chore: add back @types/jest

MarcMcIntosh committed:
chore(core): update package-lock

MarcMcIntosh committed:
chore(lerna): run unit tests in parallel

MarcMcIntosh committed:
chore: add back in @types/jest

MarcMcIntosh committed:
chore: update depcheckrc

MarcMcIntosh committed:
test(unit): run in stream mode rather than parallel

arnaudlewis committed:
Merge pull request #715 from prismicio/exs240/save-mocks

editor-mocks: add server route to save the mocks

Commit:
handle older version of nextjs

arnaudlewis committed:
apply a patch on mocks when saving a slice model instead of regen all

AlexGalays committed:
Track whether the editor was used

arnaudlewis committed:
review

arnaudlewis committed:
fix tests with new changes

arnaudlewis committed:
bump editor to the last public version

arnaudlewis committed:
Merge pull request #729 from prismicio/feat/mocks-retention

Mock retention

arnaudlewis committed:
Merge branch 'dev-mocks' into chore/bump-editor-deps

arnaudlewis committed:
Merge pull request #731 from prismicio/chore/bump-editor-deps

Bump editor version to 0.1.0 first public version

arnaudlewis committed:
Publish

 - @slicemachine/client@1.1.16-dev-mocks.0
 - @slicemachine/core@1.1.16-dev-mocks.0
 - @slicemachine/init@1.1.16-dev-mocks.0
 - slice-machine-ui@0.5.3-dev-mocks.0

arnaudlewis committed:
Merge pull request #730 from prismicio/track-editor-widget-usage

Track editor widget usage

hypervillain committed:
Merge remote-tracking branch 'origin/dev-simulator-v2' into dev-mocks-with-simv2

hypervillain committed:
fix types from build error

hypervillain committed:
fix e2e create-slice test

hypervillain committed:
Publish

 - @slicemachine/client@1.1.16-dev-mocks-with-simv2.0
 - @slicemachine/core@1.1.16-dev-mocks-with-simv2.0
 - @slicemachine/init@1.1.16-dev-mocks-with-simv2.0
 - slice-machine-ui@0.5.3-dev-mocks-with-simv2.0

hypervillain committed:
use theme light

hypervillain committed:
Merge branch 'dev-mocks-with-simv2' of github.com:prismicio/slice-machine into dev-mocks-with-simv2

danigo99 committed:
new deploy alpha script

Commit:
Merge remote-tracking branch 'origin/dev-mocks-with-simv2' into dev-sentry

Commit:
fix after merge

MarcMcIntosh committed:
refactor(init): user semver for adding the correct deps to next projects

arnaudlewis committed:
Publish

 - slice-machine-ui@0.5.3-dev-mocks.1

hypervillain committed:
fix flex height sim

hypervillain committed:
Publish

 - @slicemachine/client@1.1.16-dev-mocks-with-simv2.1
 - @slicemachine/core@1.1.16-dev-mocks-with-simv2.1
 - @slicemachine/init@1.1.16-dev-mocks-with-simv2.1
 - slice-machine-ui@0.5.3-dev-mocks-with-simv2.1

arnaudlewis committed:
Merge branch 'master' into dev-mocks

arnaudlewis committed:
Publish

 - @slicemachine/client@1.1.16-dev-0.6.0.0
 - @slicemachine/core@1.1.16-dev-0.6.0.0
 - @slicemachine/init@1.1.16-dev-0.6.0.0
 - slice-machine-ui@0.5.3-dev-0.6.0.0

hypervillain committed:
track simulator video

hypervillain committed:
iframeRenderer 2

hypervillain committed:
Merge branch 'dev-mocks-with-simv2' of github.com:prismicio/slice-machine into dev-mocks-with-simv2

hypervillain committed:
Publish

 - @slicemachine/client@1.1.16-dev-mocks-with-simv2.2
 - @slicemachine/core@1.1.16-dev-mocks-with-simv2.2
 - @slicemachine/init@1.1.16-dev-mocks-with-simv2.2
 - slice-machine-ui@0.5.3-dev-mocks-with-simv2.2

arnaudlewis committed:
Publish

 - @slicemachine/client@1.1.16-dev-mocks.1
 - @slicemachine/core@1.1.16-dev-mocks.1
 - @slicemachine/init@1.1.16-dev-mocks.1
 - slice-machine-ui@0.5.3-dev-mocks.2

arnaudlewis committed:
Merge remote-tracking branch 'origin/dev-0.6.0' into dev-mocks

arnaudlewis committed:
Merge pull request #728 from prismicio/dev-mocks

SM x Editor

Commit:
adjust sentry trace sample rate for prod

Commit:
rename sentry to pass eslint hook rule

arnaudlewis committed:
Merge branch 'dev-0.6.0' into dev-mocks-with-simv2

hypervillain committed:
UI polishing sim2

hypervillain committed:
Merge branch 'dev-mocks-with-simv2' of github.com:prismicio/slice-machine into dev-mocks-with-simv2

mathieud committed:
Publish

 - @slicemachine/client@1.1.16-dev-sentry.0
 - @slicemachine/core@1.1.16-dev-sentry.0
 - @slicemachine/init@1.1.16-dev-sentry.0
 - slice-machine-ui@0.5.3-dev-sentry.10

arnaudlewis committed:
Publish

 - @slicemachine/client@1.1.16-dev-mocks-with-simv2.3
 - @slicemachine/core@1.1.16-dev-mocks-with-simv2.3
 - @slicemachine/init@1.1.16-dev-mocks-with-simv2.3
 - slice-machine-ui@0.5.3-dev-mocks-with-simv2.3

arnaudlewis committed:
Merge pull request #734 from prismicio/dev-mocks-with-simv2

React 18 + mocks editor + simulator V2

Commit:
Merge remote-tracking branch 'origin/dev-0.6.0' into dev-sentry

Commit:
fix linting error

arnaudlewis committed:
Merge pull request #737 from prismicio/dev-sentry

Add sentry

arnaudlewis committed:
bump prismic mocks to fix random image mocks for cropper

arnaudlewis committed:
fix test with new version of prismic-mocks

arnaudlewis committed:
Publish

 - @slicemachine/client@1.1.16-dev-0.6.0.0
 - @slicemachine/core@1.1.16-dev-0.6.0.0
 - @slicemachine/init@1.1.16-dev-0.6.0.0
 - slice-machine-ui@0.5.3-dev-0.6.0.0

Commit:
clean and test code

hypervillain committed:
fix button font + sim header bg color

arnaudlewis committed:
trigger new alpha

Commit:
update other usage of function

arnaudlewis committed:
Merge pull request #741 from prismicio/simulator2-ui-polish

Simulator2 UI polish

MarcMcIntosh committed:
CI: crearte semver alpha

MarcMcIntosh committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.0
 - @slicemachine/core@1.1.16-dev-alpha.0
 - @slicemachine/init@1.1.16-dev-alpha.0
 - slice-machine-ui@0.5.3-dev-alpha.0

bapmrl committed:
done

bapmrl committed:
avoid displaying global scroll bar in simulator

bapmrl committed:
Merge branch 'dev-0.6.0-alpha' into dev-alpha

hypervillain committed:
simulator style fix

MarcMcIntosh committed:
ci: remove access to private packages

MarcMcIntosh committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.1
 - @slicemachine/core@1.1.16-dev-alpha.1
 - @slicemachine/init@1.1.16-dev-alpha.1
 - slice-machine-ui@0.5.3-dev-alpha.1

MarcMcIntosh committed:
ci: and npm token env var

MarcMcIntosh committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.2
 - @slicemachine/core@1.1.16-dev-alpha.2
 - @slicemachine/init@1.1.16-dev-alpha.2
 - slice-machine-ui@0.5.3-dev-alpha.2

MarcMcIntosh committed:
ci: make sure npm is authenticated

MarcMcIntosh committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.3
 - @slicemachine/core@1.1.16-dev-alpha.3
 - @slicemachine/init@1.1.16-dev-alpha.3
 - slice-machine-ui@0.5.3-dev-alpha.3

Commit:
leave existing function unchanged

Commit:
use simpler type

MarcMcIntosh committed:
Merge branch 'master' into dev-alpha

Conflicts:
	cypress/e2e/updates/simulator-tooltip.spec.js
	package-lock.json

On branch dev-alpha
Changes to be committed:
	new file:   cypress.config.js
	deleted:    cypress.json
	renamed:    cypress/integration/customTypes/00-create.spec.js -> cypress/e2e/customTypes/00-create.cy.js
	renamed:    cypress/integration/customTypes/01-duplicate.spec.js -> cypress/e2e/customTypes/01-duplicate.cy.js
	renamed:    cypress/integration/onboarding/00-redirects.spec.js -> cypress/e2e/onboarding/00-redirects.cy.js
	renamed:    cypress/integration/onboarding/01-skip.spec.js -> cypress/e2e/onboarding/01-skip.cy.js
	renamed:    cypress/integration/onboarding/02-finish.spec.js -> cypress/e2e/onboarding/02-finish.cy.js
	renamed:    cypress/integration/slices/00-create.spec.js -> cypress/e2e/slices/00-create.cy.js
	renamed:    cypress/integration/slices/01-duplicates.spec.js -> cypress/e2e/slices/01-duplicates.cy.js
	renamed:    cypress/integration/updates/changelog.spec.js -> cypress/e2e/updates/changelog.cy.js
	renamed:    cypress/integration/updates/sidebar.spec.js -> cypress/e2e/updates/sidebar.cy.js
	renamed:    cypress/integration/updates/simulator-tooltip.spec.js -> cypress/e2e/updates/simulator-tooltip.cy.js
	renamed:    cypress/integration/updates/video-tooltip.spec.js -> cypress/e2e/updates/video-tooltip.cy.js
	renamed:    cypress/integration/user-flows/scenario_001.spec.js -> cypress/e2e/user-flows/scenario_001.cy.js
	renamed:    cypress/support/index.js -> cypress/support/e2e.js
	modified:   package-lock.json
	modified:   package.json

Commit:
Merge pull request #727 from prismicio/chore/912-next-dependency

add new dependency when starting a next project

Commit:
use new mock format

arnaudlewis committed:
switch variation properly for simulator + screenshot

hypervillain committed:
Update sim2 video (#747)

Commit:
Merge pull request #751 from prismicio/fix/merge-problem

use new mock format

mathieud committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.4
 - @slicemachine/core@1.1.16-dev-alpha.4
 - @slicemachine/init@1.1.16-dev-alpha.4
 - slice-machine-ui@0.5.3-dev-alpha.4

arnaudlewis committed:
Merge branch 'dev-alpha' into fix/variations-switch

bapmrl committed:
fix semver's circular dep

bapmrl committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.5
 - @slicemachine/core@1.1.16-dev-alpha.5
 - @slicemachine/init@1.1.16-dev-alpha.5
 - slice-machine-ui@0.5.3-dev-alpha.5

dani-mp committed:
Change copy in edit modal

dani-mp committed:
Add DeprecatedMockConfigMessage

dani-mp committed:
Delete Image Mock

hypervillain committed:
remove iframe check

Commit:
add sentry token to alpha deploy flow

arnaudlewis committed:
Merge pull request #752 from prismicio/fix/variations-switch

Fix: Switch variation on simulator

arnaudlewis committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.6
 - @slicemachine/core@1.1.16-dev-alpha.6
 - @slicemachine/init@1.1.16-dev-alpha.6
 - slice-machine-ui@0.5.3-dev-alpha.6

arnaudlewis committed:
improve how we manage the editor state in simulator

MarcMcIntosh commented on .github/workflows/test-and-deploy-alpha.yml:
```diff
@@ -104,3 +104,31 @@ jobs:
+    steps: 
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/setup-node
+      - name: Install Dependencies
```
do we need to do this twice?

MarcMcIntosh committed:
wip: change when tab id is trimmed

hypervillain committed:
re-add retry

hypervillain committed:
trck fail once

arnaudlewis committed:
fix richtext re-render issue

Commit:
set editor width to 33% when open

Commit:
fix troubleshooting link

Commit:
Merge pull request #758 from prismicio/fix/editor-width

set editor width to 33% when open

mathieud committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.7
 - @slicemachine/core@1.1.16-dev-alpha.7
 - @slicemachine/init@1.1.16-dev-alpha.7
 - slice-machine-ui@0.5.3-dev-alpha.7

Commit:
fix editor width

Commit:
Merge pull request #759 from prismicio/fix/editor-width

fix editor width

mathieud committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.8
 - @slicemachine/core@1.1.16-dev-alpha.8
 - @slicemachine/init@1.1.16-dev-alpha.8
 - slice-machine-ui@0.5.3-dev-alpha.8

Heyrows committed:
fix header for slices for small devices

MarcMcIntosh committed:
test(simulator): unit for saving a mock

hypervillain committed:
remove retry + add refresh

hypervillain committed:
Merge branch 'fix/iframe-retry' of github.com:prismicio/slice-machine into fix/iframe-retry

hypervillain committed:
rm triggerReload from iframerenderer

MarcMcIntosh committed:
test(simulator): remove extended timeout

arnaudlewis committed:
remove libraries-state.json from simulator templates

bapmrl committed:
fix wrongly modified comment

bapmrl committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.9
 - @slicemachine/core@1.1.16-dev-alpha.9
 - @slicemachine/init@1.1.16-dev-alpha.9
 - slice-machine-ui@0.5.3-dev-alpha.9

dani-mp committed:
Improve image mock tab copy

dani-mp committed:
Merge pull request #755 from prismicio/dani/image-mock

Deprecate image mock config

AlexGalays committed:
Rename event

dani-mp committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.10
 - @slicemachine/core@1.1.16-dev-alpha.10
 - @slicemachine/init@1.1.16-dev-alpha.10
 - slice-machine-ui@0.5.3-dev-alpha.10

MarcMcIntosh committed:
fix(save mocks): update selected slice after saving mock

MarcMcIntosh committed:
chore(simulator): update test for saga

MarcMcIntosh committed:
refactor(selectedSlice): return null if there is no selected slice

dani-mp committed:
Remove ImageLegacyMockConfig

dani-mp committed:
Upgrade the mocks library

Heyrows committed:
refactor header component, use it on slice builder page

Heyrows committed:
remove useless MR

MarcMcIntosh committed:
test(e2e: simulator): add test for saving variations

MarcMcIntosh committed:
chore: run prettier

dani-mp committed:
Fix tests related to the image mock config

danigo99 committed:
fix icon fills, simualtor border and logo size

dani-mp committed:
Merge pull request #767 from prismicio/dani/image-mock

dani-mp committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.11
 - @slicemachine/core@1.1.16-dev-alpha.11
 - @slicemachine/init@1.1.16-dev-alpha.11
 - slice-machine-ui@0.5.3-dev-alpha.11

MarcMcIntosh committed:
test(update tab): add e2e test

MarcMcIntosh committed:
chore: run pretty

MarcMcIntosh committed:
refactor(selectedSlice reducer): remove unneeded property

MarcMcIntosh committed:
Merge pull request #757 from prismicio/fix/editor-state

Improve editor state management

MarcMcIntosh committed:
Merge pull request #770 from prismicio/sm876/whitespace-tab

Update tab trimming whitespace

bapmrl committed:
fix useThrottle

danigo99 committed:
Fetch changelog only on app restart

danigo99 committed:
Merge pull request #769 from prismicio/fix/simulator-ui-bugs

fix icon fills, simualtor border and logo size

danigo99 committed:
Fix sentry setupt after remove promise from getEnv

MarcMcIntosh committed:
test(e2e): not sure why this one is failing

MarcMcIntosh committed:
chore: run pretty

MarcMcIntosh committed:
e2e: support function maybe this works :/

bapmrl committed:
fix editor width and iframe border

MarcMcIntosh committed:
chore: run prettier

MarcMcIntosh committed:
Merge branch 'dev-alpha' into fix/iframe-retry

 Conflicts:
	packages/slice-machine/components/Simulator/index.tsx

 Changes to be committed:
	modified:   .github/workflows/test-and-deploy-alpha.yml
	modified:   cypress/e2e/customTypes/00-create.cy.js
	modified:   cypress/e2e/updates/simulator-tooltip.cy.js
	modified:   cypress/e2e/updates/video-tooltip.cy.js
	modified:   cypress/support/commands.js
	modified:   cypress/support/index.d.ts
	modified:   packages/client/package-lock.json
	modified:   packages/client/package.json
	modified:   packages/core/package-lock.json
	modified:   packages/core/package.json
	modified:   packages/init/package-lock.json
	modified:   packages/init/package.json
	modified:   packages/init/src/utils/versionCheck.ts
	modified:   packages/slice-machine/components/Simulator/components/FailedConnect/index.tsx
	modified:   packages/slice-machine/components/Simulator/index.tsx
	modified:   packages/slice-machine/lib/builders/CustomTypeBuilder/TabModal/update.tsx
	modified:   packages/slice-machine/lib/builders/common/EditModal/index.js
	modified:   packages/slice-machine/lib/mock/LegacyMockConfig.ts
	new file:   packages/slice-machine/lib/models/common/DeprecatedMockConfigMessage.tsx
	deleted:    packages/slice-machine/lib/models/common/widgets/Image/Mock/Form.js
	deleted:    packages/slice-machine/lib/models/common/widgets/Image/Mock/components.js
	deleted:    packages/slice-machine/lib/models/common/widgets/Image/Mock/dataset/Animal.js
	deleted:    packages/slice-machine/lib/models/common/widgets/Image/Mock/dataset/Architecture.js
	deleted:    packages/slice-machine/lib/models/common/widgets/Image/Mock/dataset/Art.js
	deleted:    packages/slice-machine/lib/models/common/widgets/Image/Mock/dataset/Business.js
	deleted:    packages/slice-machine/lib/models/common/widgets/Image/Mock/dataset/Colorful.js
	deleted:    packages/slice-machine/lib/models/common/widgets/Image/Mock/dataset/Computer.js
	deleted:    packages/slice-machine/lib/models/common/widgets/Image/Mock/dataset/Minimal.js
	deleted:    packages/slice-machine/lib/models/common/widgets/Image/Mock/dataset/Nature.js
	deleted:    packages/slice-machine/lib/models/common/widgets/Image/Mock/dataset/Portrait.js
	deleted:    packages/slice-machine/lib/models/common/widgets/Image/Mock/dataset/index.js
	modified:   packages/slice-machine/lib/models/common/widgets/Image/index.ts
	modified:   packages/slice-machine/lib/models/common/widgets/Widget.ts
	modified:   packages/slice-machine/package-lock.json
	modified:   packages/slice-machine/package.json
	modified:   packages/slice-machine/tests/lib/mock/LegacyMockConfig.test.ts
	modified:   packages/slice-machine/tests/lib/mock/Slice.test.ts

MarcMcIntosh committed:
test(e2e): user helper function for getting by label.

hypervillain committed:
fix: iframe retry (#756)

MarcMcIntosh committed:
e2e: refactor tests

MarcMcIntosh committed:
e2e: refactor support commands

MarcMcIntosh committed:
chore: run pretty

MarcMcIntosh committed:
e2e: refactor clicking button

MarcMcIntosh committed:
Merge branch 'dev-alpha' into fix/iframe-retry

 Conflicts:
	cypress/e2e/slices/00-create.cy.js

	modified:   cypress/e2e/customTypes/00-create.cy.js

MarcMcIntosh committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.12
 - @slicemachine/core@1.1.16-dev-alpha.12
 - @slicemachine/init@1.1.16-dev-alpha.12
 - slice-machine-ui@0.5.3-dev-alpha.12

Heyrows committed:
remove unused ref

danigo99 committed:
Merge branch 'dev-alpha' into fetch-changelog

Heyrows committed:
fix unit tests

Heyrows committed:
Merge branch 'dev-alpha' into 933/fix-header-small-design

Heyrows committed:
Merge pull request #761 from prismicio/933/fix-header-small-design

fix header for slices for small devices

Heyrows committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.13
 - @slicemachine/core@1.1.16-dev-alpha.13
 - @slicemachine/init@1.1.16-dev-alpha.13
 - slice-machine-ui@0.5.3-dev-alpha.13

danigo99 committed:
Merge pull request #771 from prismicio/fetch-changelog

Fetch changelog only on app restart

Heyrows committed:
Merge pull request #763 from prismicio/fix/simulator-templates

Fix simulator templates

Heyrows committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.14
 - @slicemachine/core@1.1.16-dev-alpha.14
 - @slicemachine/init@1.1.16-dev-alpha.14
 - slice-machine-ui@0.5.3-dev-alpha.14

Commit:
disable simulator button when slice changed

danigo99 committed:
WIP 2

Commit:
display tooltip on disabled button

AlexGalays committed:
Merge branch 'dev-alpha' into alex/rename-editor-widget-used-event

Commit:
fix onboarding video

Commit:
fix tooltip bug

when closing the main video tooltip while the simulator one was also open
bogh tooltips would close
the ref to the main tooltip was null at the moment the closing function ran

Commit:
fix missing key warning

danigo99 committed:
Disable save mock button when simulator fails to load

AlexGalays committed:
Add sliceId to Editor Widget Used event

danigo99 committed:
Fix header padding

AlexGalays committed:
Merge pull request #766 from prismicio/alex/rename-editor-widget-used-event

AlexGalays committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.15
 - @slicemachine/core@1.1.16-dev-alpha.15
 - @slicemachine/init@1.1.16-dev-alpha.15
 - slice-machine-ui@0.5.3-dev-alpha.15

danigo99 committed:
Merge pull request #775 from prismicio/948/disable-save-mock

Disable save mock button when there is a problem loading the simulator and fix header

danigo99 committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.16
 - @slicemachine/core@1.1.16-dev-alpha.16
 - @slicemachine/init@1.1.16-dev-alpha.16
 - slice-machine-ui@0.5.3-dev-alpha.16

Commit:
update tooltip content

MarcMcIntosh committed:
Update dependency: @prismicio/slice-simulator-com@0.4.1

@prismicio/slice-simulator-com
  * slice-machine-ui: ^0.4.1-alpha.0 → 0.4.1

MarcMcIntosh committed:
Merge pull request #777 from prismicio/mm/update-simulator-common

Update dependency: @prismicio/slice-simulator-com@0.4.1

Commit:
add missing hook dependencies

Commit:
Merge pull request #773 from prismicio/fix/simulator-crash

Fix/simulator crash

mathieud committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.17
 - @slicemachine/core@1.1.16-dev-alpha.17
 - @slicemachine/init@1.1.16-dev-alpha.17
 - slice-machine-ui@0.5.3-dev-alpha.17

Commit:
apply same color to the icon when disabled

MarcMcIntosh committed:
Merge pull request #772 from prismicio/fix/iframe-retry

Fix iframe retry

MarcMcIntosh committed:
Merge pull request #778 from prismicio/fix/double-braces

Fix double braces in uuid field

MarcMcIntosh committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.18
 - @slicemachine/core@1.1.16-dev-alpha.18
 - @slicemachine/init@1.1.16-dev-alpha.18
 - slice-machine-ui@0.5.3-dev-alpha.18

Commit:
Merge pull request #780 from prismicio/fix/disabled-icon-color

apply same color to the icon when disabled

mathieud committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.19
 - @slicemachine/core@1.1.16-dev-alpha.19
 - @slicemachine/init@1.1.16-dev-alpha.19
 - slice-machine-ui@0.5.3-dev-alpha.19

bapmrl committed:
fix: iframe scaling (#776)

bapmrl committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.20
 - @slicemachine/core@1.1.16-dev-alpha.20
 - @slicemachine/init@1.1.16-dev-alpha.20
 - slice-machine-ui@0.5.3-dev-alpha.20

bapmrl committed:
TDD 😘

bapmrl committed:
update @prismicio/mocks

danigo99 committed:
add cursor not-allowed to screenshot button disabled state

bapmrl committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.21
 - @slicemachine/core@1.1.16-dev-alpha.21
 - @slicemachine/init@1.1.16-dev-alpha.21
 - slice-machine-ui@0.5.3-dev-alpha.21

danigo99 committed:
Update latest version in changelog page when the versions change

danigo99 committed:
Merge pull request #782 from prismicio/949/disable-screenshot-button

add cursor not-allowed to screenshot button disabled state

danigo99 committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.22
 - @slicemachine/core@1.1.16-dev-alpha.22
 - @slicemachine/init@1.1.16-dev-alpha.22
 - slice-machine-ui@0.5.3-dev-alpha.22

bapmrl committed:
Merge branch 'master' into dev-alpha

bapmrl committed:
Merge branch 'dev-alpha' of github.com:prismicio/slice-machine into dev-alpha

bapmrl committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.23
 - @slicemachine/core@1.1.16-dev-alpha.23
 - @slicemachine/init@1.1.16-dev-alpha.23
 - slice-machine-ui@0.5.3-dev-alpha.23

danigo99 committed:
Disable other button effects when disabled

danigo99 committed:
Merge pull request #784 from prismicio/fix/buttons-disabled-state

Disable other button effects when disabled

danigo99 committed:
use latest version in useEffect hook

danigo99 committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.24
 - @slicemachine/core@1.1.16-dev-alpha.24
 - @slicemachine/init@1.1.16-dev-alpha.24
 - slice-machine-ui@0.5.3-dev-alpha.24

danigo99 committed:
Add loading spinner when changelog page is loading

danigo99 committed:
move null to use state

danigo99 committed:
Merge pull request #783 from prismicio/916/fix/update-cl-page

Update latest version in changelog page when the versions change

danigo99 committed:
Publish

 - @slicemachine/client@1.1.16-dev-alpha.25
 - @slicemachine/core@1.1.16-dev-alpha.25
 - @slicemachine/init@1.1.16-dev-alpha.25
 - slice-machine-ui@0.5.3-dev-alpha.25
